plugins/modules/site_workflow_manager.py

```diff
@@ -1304,18 +1244,19 @@ def validate_site_input_data(self, config):
 
             if site_type == "floor":
                 self.log("Performing floor-specific validations.", "DEBUG")
-                floor_number = site.get(site_type, {}).get("floor_number")
-                if floor_number:
-                    self.log("Validating 'floor_number': " + str(floor_number), "DEBUG")
-                    if not (isinstance(floor_number, int) and -200 <= floor_number <= 200):
-                        errormsg.append("Please enter a valid floor number (-200 to 200)")
-                        self.log("Missing 'floor_number' in floor entry.", "ERROR")
-                else:
-                    errormsg.append("Floor number should not be None or empty")
+                if self.compare_dnac_versions(self.get_ccc_version(), "2.3.7.6") >= 0:
+                    floor_number = site.get(site_type, {}).get("floor_number")
+                    if floor_number:
+                        self.log("Validating 'floor_number': " + str(floor_number), "DEBUG")
+                        if not (isinstance(floor_number, int) and -200 <= floor_number <= 200):
+                            errormsg.append("Please enter a valid floor number (-200 to 200)")
+                            self.log("Missing 'floor_number' in floor entry.", "ERROR")
```

Comment:
self.log("'floor_number' is out of the valid range (-200 to 200).", "ERROR")

Reply:
Modified